src/components/layout/sidebar/Menu.jsx

```diff
@@ -14,6 +14,7 @@ class Menu extends React.Component {
       <div>
         <Link to="/" style={styles.logo}>Coral Project</Link>
         <ul>
+          <MenuItem name="Tag Manager" target="/tag-manager" />
```

Comment:
@jde wanted to change the name here to "Settings" for now. We'll make this it's own route later. 4th item on the issue [here](https://github.com/coralproject/cay/issues/43)